applications/GeoMechanicsApplication/tests/cpp_tests/custom_processes/test_apply_c_phi_reduction_process.cpp

```diff
@@ -200,4 +208,23 @@ KRATOS_TEST_CASE_IN_SUITE(CheckFailureNegativeReductionFactorApplyCPhiReductionP
         "Reduction factor should not drop below 0.01, calculation stopped.");
 }
 
+KRATOS_TEST_CASE_IN_SUITE(CheckFailureTooSmallReductionIncrementApplyCPhiReductionProcess, KratosGeoMechanicsFastSuite)
+{
+    Model model;
+    auto& r_model_part = PrepareCPhiTestModelPart(model);
+    // Number of cycles should be larger than one to trigger the step halving of the
+    // reduction increment
+    r_model_part.GetProcessInfo().GetValue(NUMBER_OF_CYCLES) = 10;
+    ApplyCPhiReductionProcess process{r_model_part, {}};
+    for (size_t i = 0; i < 6; ++i) {
+        process.ExecuteInitializeSolutionStep();
+    }
+
+    // After halving the initial reduction increment (0.1) for the seventh time, it becomes
+    // 0,00078125, so it should throw an exception
```

Comment:
Nitpicking (use the dot as decimal separator):
```suggestion
    // 0.00078125, so it should throw an exception
```